File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Concepts/ClientAPI/ListBox/ListBoxPage.vb
Namespace CompanionKit.Concepts.ClientAPI.ListBox

	Public Class ListBoxPage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

	        

		End Sub

        ''' <summary>
        ''' Handles the Click event of the mobjFillButton control.
        ''' </summary>
        ''' <param name="sender">The source of the event.</param>
        ''' <param name="e">The <see cref="System.EventArgs"/> instance containing the event data.</param>
        Private Sub mobjFillButton_Click(sender As System.Object, e As System.EventArgs) Handles mobjFillButton.Click
            VWGClientContext.Current.Invoke("fillListBox")
        End Sub

        ''' <summary>
        ''' Handles the Click event of the mobjClearButton control.
        ''' </summary>
        ''' <param name="sender">The source of the event.</param>
        ''' <param name="e">The <see cref="System.EventArgs"/> instance containing the event data.</param>
        Private Sub mobjClearButton_Click(sender As System.Object, e As System.EventArgs) Handles mobjClearButton.Click
            VWGClientContext.Current.Invoke("clearListBox")
        End Sub
    End Class

End Namespace

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Concepts/ClientAPI/ListBox/ListBoxPage.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;

namespace CompanionKit.Concepts.ClientAPI.ListBox
{
    public partial class ListBoxPage : UserControl
    {
        public ListBoxPage()
        {
            InitializeComponent();
            
        }


        /// <summary>
        /// Handles the Click event of the mobjFillButton control.
        /// </summary>
        /// <param name="sender">The source of the event.</param>
        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        private void mobjFillButton_Click(object sender, EventArgs e)
        {
            VWGClientContext.Current.Invoke("fillListBox");
        }

        /// <summary>
        /// Handles the Click event of the mobjClearButton control.
        /// </summary>
        /// <param name="sender">The source of the event.</param>
        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        private void mobjClearButton_Click(object sender, EventArgs e)
        {
            VWGClientContext.Current.Invoke("clearListBox");
        }



    }
}